test/backward_compatibility/check_backward_compatibility.py

```diff
@@ -132,6 +132,12 @@
     ("aten::add_relu", datetime.date(2020, 10, 28)),
     ("aten::add_relu_", datetime.date(2020, 10, 28)),
     ("aten::hash", datetime.date(2020, 11, 15)),
+    ("__caffe2::RoIAlign", datetime.date(2020, 11, 15)),
+    ("__caffe2::HeatmapMaxKeypoint", datetime.date(2020, 11, 15)),
+    ("__caffe2::BoxWithNMSLimit", datetime.date(2020, 11, 15)),
+    ("__caffe2::BBoxTransform", datetime.date(2020, 11, 15)),
+    ("__caffe2::GenerateProposals", datetime.date(2020, 11, 15)),
+    ("__caffe2::RoIAlignRotated", datetime.date(2020, 11, 15)),
```

Comment:
just calling out the BC-breakingness again here. each of these ops is registered twice under both the `_caffe` and `__caffe2` namespaces. I couldn't find any usages of the latter version of these 5 ops from a grep search, and I think it makes more sense to hardcode the `_caffe2` namespace directly into the macro, so I opted to remove these 5 ops here. If there's a reason that you think they should still be around, let me know.